lte/gateway/python/magma/pipelined/app/policy_mixin.py

```diff
@@ -164,7 +164,7 @@ def _get_classify_rule_flow_msgs(self, imsi, msisdn: bytes, uplink_tunnel: int,
                     passthrough_actions,
                     hard_timeout=hard_timeout,
                     priority=Utils.UNCLASSIFIED_ALLOW_PRIORITY,
-                    cookie=rule_num,
+                    cookie=shard_id,
```

Comment:
I think lets not modify cookie here, we don't care about other tables for polling shards so lets keep
cookie=rule_num in this file